Dockerfile

```diff
@@ -94,6 +94,14 @@ ENV ZIP_SUBDIR true
 ENV LOGS_SUBDIR true
 
 # Apply the MicroG's signature spoofing patch
+# Since LineageOS 21, signature spoofing for microG is built into LineageOS
+# itself and no patches are required if you're building an userdebug or eng
+# variant. In this case, this value can be left set to "no".
+# You may set the value to "restricted" if you wish to build an user variant
+# with signature spoofing enabled; otherwise, it will be disabled in user
+# builds.
+#
+# For LineageOS 20 and older:
```

Comment:
LOS has added signature spoofing to 18.1, 19.1, 20.0 and 21.0. L4M's patches just aren't broken on versions below 21 AFAIK.

Reply:
Oh, I didn't realize older LOS versions also now have built-in signature spoofing. Will change documentation to reflect this.

Reply:
You could also add the same patches for removing the debug check for those versions, the code should be the same.